library/src/nvidia_detail/hipsolver_dense64.cpp

```diff
@@ -192,4 +192,69 @@ catch(...)
     return hipsolver::exception2hip_status();
 }
 
+/******************** POTRF ********************/
+hipsolverStatus_t hipsolverDnXpotrf_bufferSize(hipsolverDnHandle_t handle,
+                                               hipsolverDnParams_t params,
+                                               hipsolverFillMode_t uplo,
+                                               int64_t             n,
+                                               hipDataType         dataTypeA,
+                                               const void*         A,
+                                               int64_t             lda,
+                                               hipDataType         computeType,
+                                               size_t*             lworkOnDevice,
+                                               size_t*             lworkOnHost)
+try
+{
+    if(!handle)
+        return HIPSOLVER_STATUS_NOT_INITIALIZED;
+    if(!params)
+        return HIPSOLVER_STATUS_INVALID_VALUE;
+
+    return hipsolver::cuda2hip_status(cusolverDnXpotrf_bufferSize(
+        handle, params, uplo, n, dataTypeA, A, lda, computeType, lworkOnDevice, lworkOnHost));
```

Comment:
I am still not completely familiar with hipSOLVER but I thought that `handle` and `params` were meant to be cast to types `cusolverDnHandle_t` and `cusolverDnParams_t`. Also, `cusolverDnXpotrf_bufferSize` should probably expect the result of `hipsolver::hip2cuda_operation(uplo)` (or something similar) instead of a `uplo` (which has type `hipsolverFillMode_t`).